src/js/components/Calendar/doc.js

```diff
@@ -41,6 +41,9 @@ export const doc = Calendar => {
       ]),
     ).description(`Multiple dates in ISO8601 format that should not be
         selectable. Items that are an array indicate a range of dates.`),
+    daysOfWeek: PropTypes.bool
+      .description(`Whether to show the days of the week.`)
+      .defaultValue(true),
```

Comment:
I think the implementation default is `false`, which it has to be to not be a breaking change.